website/docs/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -113,7 +113,7 @@ curl \
         "oauth-tokens": {
           "data": [],
           "links": {
-            "related": "/api/v2/oauth-tokens/ot-KaeqH4cy72VPXFQT"
+            "related": "/api/v2/oauth-clients/oc-XKFwG6ggfA9n7t1K/oauth-tokens"
```

Comment:
Thank you for catching this! Updating to make this a bit more vague, but clearer:
```suggestion
            "related": "/api/v2/oauth-clients/<OAUTH_CLIENT_ID>/oauth-tokens"
```